tests/test_glfw_dropfile.c

```diff
@@ -0,0 +1,87 @@
+#include <stdio.h>
+#include <assert.h>
+#ifdef __EMSCRIPTEN__
+#include <emscripten.h>
+#endif
+#define GLFW_INCLUDE_ES2
+#include <GLFW/glfw3.h>
+
+int result = 1;
+
+GLFWwindow* g_window;
+
+void render();
+
+void render() {
+  glClearColor(0.0f, 1.0f, 0.0f, 1.0f);
+  glClear(GL_COLOR_BUFFER_BIT);
+}
+
+void on_file_drop(GLFWwindow *window, int count, const char **paths) {
+  for (int i = 0; i < count; ++i) {
+    printf("dropped file %s\n", paths[i]);
+
+    FILE *fp = fopen(paths[i], "rb");
+    if (!fp) {
+        printf("failed to open %s\n", paths[i]);
+        perror("fopen");
+        result = 0;
+        continue;
+    }
+    int c;
+    long size = 0;
+    while ((c = fgetc(fp) != -1)) {
+        ++size;
+    }
+    printf("read %ld bytes from %s\n", size, paths[i]);
```

Comment:
Great test! This does not actually do anything with the contents of the file though, so perhaps it would be good to print out e.g. 10 first and 10 last characters of the file, if one drops a text file, so it'll allow verifying that the actual bytes were transmitted correctly.